requirements/typing.in

```diff
@@ -0,0 +1,2 @@
+mypy
+pytest
```

Comment:
Don't need to add pytest since we're not typing the tests.

Reply:
Ta